doc/aaa/radius_authentication.md

```diff
@@ -0,0 +1,634 @@
+                                                 implementing fallback to local.
+- This is an existing table specification, which has been updated for RADIUS
+
+### RADIUS Table Schema
+
+```
+; RADIUS configuration attributes global to the system. Only one instance of the table exists in the system. Any of the global RADIUS attributes can be overwritten by the per server settings.
+; Key
+global_key           = "global"  ;  RADIUS global configuration
+; Attributes
+passkey              = 1*32VCHAR  ; shared secret (Valid chars: ASCII printable except SPACE, '#', and COMMA)
+auth_type            = "pap" / "chap" / "mschapv2"  ; method used for authenticating the communication message
+src_ip               = IPAddress  ;  source IP address (IPv4 or IPv6) for the
+                                  ;  outgoing UDP/IP dgram. This is being
+                                  ;  obsoleted in favor of the RADIUS_SERVER
+                                  ;  table src_intf attribute. If both src_ip
+                                  ;  and src_intf are specified a warning log
+                                  ;  can be given, and src_intf is preferred.
+                                  ;  Default is determined by routing stack.
+nas_ip               = IPAddress  ;  NAS-IP|IPV6-Address (Type 4|95) attribute
+                                  ;  in the outgoing RADIUS PDU.
+                                  ;  Default is to use an IPAddress from the
+                                  ;  MGMT_INTERFACE table.
+statistics           = "True" / "False" ;  Enable statistics collection
+timeout              = 1*2DIGIT
+retransmit           = 1*2DIGIT
+vrf                  = vrf_name  ; Use default vrf if not specified
+                                 ; (WIP) RADIUS_SERVER attribute is preferred.
+src_intf             = source_interface ; Eg: eth0, Loopback0... Use the first
+                                 ; IPAddress retrieved for interface for the
+                                 ; outgoing UDP/IP dgram. Default is
+                                 ; determined by routing stack
+                                 ; (WIP) RADIUS_SERVER attribute is preferred.
+```
+
+
+### RADIUS_SERVER Table Schema
+
+```
+; RADIUS per server configuration in the system.
+; Key
+server_key           = NameOrIPAddress; RADIUS server's DNS name or IPv4|6 addr
+                                      ; * DNS name is Work In Progress(WIP) *
+                                      ; * in future phase                   *
+; Attributes
+auth_port            = 1*5DIGIT
+passkey              = 1*32VCHAR  ; per server shared secret (Valid chars: ASCII printable except SPACE, '#', and COMMA)
+auth_type            = "pap" / "chap" / "mschapv2"  ; method used for authenticating the communication message
+priority             = 1*2DIGIT  ; specify RADIUS server's priority
+timeout              = 1*2DIGIT
+retransmit           = 1*2DIGIT
```

Comment:
Can you please provide some details on retransmit ?
Part of UDP config?

Reply:
I will update as per the documentation of the pam_radius_auth module.

```
retry = #      - How many times to re-send a packet if there is no
                 response.  Once the retry count has been reached,
                 the module fails, and PAM continues to the next module.
```